yarn-project/validator-client/src/config.ts

```diff
@@ -31,9 +31,15 @@ export const validatorClientConfigMappings: ConfigMappingsType<ValidatorClientCo
   validatorPrivateKeys: {
     env: 'VALIDATOR_PRIVATE_KEYS',
     description: 'List of private keys of the validators participating in attestation duties',
-    ...secretValueConfigHelper<`0x${string}`[]>(val =>
-      val ? val.split(',').map<`0x${string}`>(key => `0x${key.replace('0x', '')}`) : [],
-    ),
+    ...secretValueConfigHelper<`0x${string}`[]>(val => {
+      if (val && val.length > 0) {
```

Comment:
Is this not only run if `VALIDATOR_PRIVATE_KEYS` is specified?

Reply:
ah yes indeed and I wasn't aware of the new `fallback` that was added, makes it a lot simpler